spec/routes/api/vm_spec.rb

```diff
@@ -43,55 +92,193 @@
         get "/api/project/#{project.ubid}/location/#{TEST_LOCATION}/vm"
 
         expect(last_response.status).to eq(200)
-        expect(JSON.parse(last_response.body)).to eq([])
+        expect(JSON.parse(last_response.body)["values"]).to eq([])
+        expect(JSON.parse(last_response.body)["next_cursor"]).to be_nil
+        expect(JSON.parse(last_response.body)["count"]).to eq(0)
       end
 
-      it "success" do
+      it "success single" do
```

Comment:
Tests for pagination features like 'success single', 'success multiple', and 'success order column' will be the same for all resources. Is there a way to reduce duplication? @fdr might have an idea

Reply:
I've moved paginated related tests to its' own file, keeping tests here covering only vm API codes.